### ISSUE TITLE
"status badge" not displaying properly

### ISSUE THREAD

**Issue by rempsyc** [easystats/easystats#425]
It seems that "status badge" not displaying properly on any of the easystats sites using it (`effectsize`, `bayestestR`, `correlation`, `insight`, `modelbased`, `performance`, `parameters`, and `report`).

![image](https://github.com/easystats/easystats/assets/13123390/06fce557-047f-4c39-b219-ea9fd8a03061)

For `effectsize` for example it refers to https://tinyverse.netlify.com/badge/effectsize/, but the site appears to be down:

![image](https://github.com/easystats/easystats/assets/13123390/63220668-016e-4988-b11d-daf46493bdff)

Indeed, https://tinyverse.netlify.com/ doesn't seem to be a valid URL anymore. I can't remember what this badge was supposed to showcase, but should we remove it or replace it for all sites then?

**Comment by etiennebacher** [comment]
The correct URL is now `https://tinyverse.netlify.app/badge/<pkg>`

**Comment by rempsyc** [comment]
Ok thanks Etienne! But in any case, even if we can fix the badge--do we actually want this badge our own pgkdown sites? Showing dependencies? Since it only shows core dependencies, so internal to `easystats`.

**Comment by IndrajeetPatil** [comment]
At some point, pkgdown website should itself show [the deps](https://github.com/r-lib/pkgdown/pull/2520), and then this would become even more redundant.

**Comment by strengejacke** [comment]
While we're on it, do we need the lifecycle badge? From the website:

> Maturing
> Previously we used as maturing for functions that lay somewhere between experimental and stable. We stopped using this stage because, like questioning, it’s not clear what actionable information this stage delivers.

**Comment by IndrajeetPatil** [comment]
Yeah, I agree that its value proposition is a bit suspect. Let's get rid of that badge as well.

We already follow semantic versioning so users already have some idea about lifecycle of our projects.

**Comment by rempsyc** [comment]
I've removed both the status and lifecycle badges from `report`

**Comment by rempsyc** [comment]
Should we close this issue and/or create a checklist for all packages to remove them?

**Comment by IndrajeetPatil** [comment]
SGTM. Thanks for taking care of it!
